test/array.test.ts

```diff
@@ -2,7 +2,8 @@ import { dedent } from 'ts-dedent'
 import ts from 'typescript'
 import { describe, expect, it } from 'vitest'
 import { z } from 'zod'
-import { printNode, zodToTs } from '../src'
+import { zodToTs } from '../src'
+import { printNodeTest } from './utils.js'
```

Comment:
```suggestion
import { printNodeTest } from './utils'
```
should be changed everywhere